sdk/python/pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "hatchling.build"
 
 [project]
 name = "pulumi"
-version = "3.165.0"
+version = "3.166.0"
```

Comment:
I think this might be missing the version in `uv.lock`? Normally the script does this for us https://github.com/pulumi/pulumi/blob/d96713f40c4c2f17b3880e09f7234a40715c69e8/scripts/set-version.py#L43

Reply:
Yeah, I missed adding that, my bad.